README.md

```diff
@@ -64,6 +50,32 @@ logger.close().then(() => {
 });
 ```
 
+### Authentication
+ELKKinesisLogger uses a credential provider chain, loading credentials from:
```

Comment:
Perhaps we should allow the caller to pass a credential chain they have constructed?

Reply:
That's a great idea! I'll do that as a follow up PR.